src/modifiers.rs

```diff
@@ -1,4 +1,5 @@
-//! Defines a series of convenience modifiers for editing Responses
+//! This module defines a series of convenience modifiers for editing
```

Comment:
I might use 'change' rather than 'edit' here, for consistency with the rest of everything.